storage/bud/records.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Since I've gotten a few questions about this:

I did intend to add this file to version control. This file is used by `@roots/bud-caching` to determine if modules in a build have changed since the build was last run. If the modules are still valid (there have been no changes to the config file or the changes to the config file did not modify the utilized modules) then it will skip needlessly recompiling those modules altogether.

In practice, this means that once this file is generated build time is significantly truncated. Further, Bud goes out of its way to not use absolute paths for resolving modules, so these path should be equally valid for development and CI. By keeping this file in version control Bud can utilize it in the context of a remote build and save time there, as well. 

If we don't want to keep this file in the Sage repo because it's super ugly, I appreciate that, but we should encourage users to not add this file to their .gitignore if they use a CI service.